### ISSUE TITLE
Community chat?

### ISSUE THREAD

**Issue by WhiteBlackGoose** [icedland/iced#219]
Hey.

This is absolutely awesome project. Made it so easy to encode/decode asm! So... 

Wouldn't it a good idea to have a community chat for Iced? So that people could ask questions here, talk, help each other, discuss the project.

**Comment by wtfsck** [comment]
I just enabled the discussions tab which can be used for that: https://github.com/icedland/iced/discussions

**Comment by WhiteBlackGoose** [comment]
That's great, but it isn't really a chat, more like a forum.

**Comment by wtfsck** [comment]
Yeah that's all they got. I want to keep everything here at GitHub. Code, CI, issues, PRs, etc.

**Comment by WhiteBlackGoose** [comment]
Sure thing. All I suggested is having a separate (Discord) chat for talking, where you can quickly ask about something, or just say "thanks" to the library authors, or just some offtopic talks... But all bugs, proposals, PRs are still here, at GH.

It's a common practice for projects, for example, [terrafx](https://github.com/terrafx/terrafx), [Silk.NET](https://github.com/dotnet/Silk.NET), [Plotly.NET](https://github.com/plotly/Plotly.NET), [.NET](https://github.com/dotnet/runtime), [AngouriMath](https://github.com/asc-community/AngouriMath) - all those and more use Discord servers for chatting. [F#](https://github.com/dotnet/fsharp), [runtimelab](https://github.com/dotnet/runtimelab/tree/feature/NativeAOT), and more are on slack and gitter.

**Comment by Kixiron** [comment]
I believe the discussions tab got disabled, that's really useful for questions (as opposed to change requests, bugs, etc.)

**Comment by wtfsck** [comment]
Re-enabled it again. ;)
